client/settings/general-settings/style.scss

```diff
@@ -0,0 +1,42 @@
+.general-settings {
+	&__bank-information {
+		display: flex;
+		flex-direction: column;
+
+		> :first-child {
+			flex-grow: 1;
+		}
+
+		@include breakpoint( '>800px' ) {
+			align-items: center;
+			justify-content: space-between;
+			flex-direction: row;
+			flex-wrap: nowrap;
+		}
+	}
+
+	&__account-statement-wrapper {
+		position: relative;
+
+		input[maxlength='22'] {
```

Comment:
```suggestion
		.general-settings__account-statement-input {
```

Reply:
I was going back and forth on this - on one hand I preferred not using the class names added by the component (since they are pretty specific and we need both `components-base-control__field` and `components-text-control__input`) and preferred the `[maxLength]` selector (since it's less likely to change and needed less class names), on the other hand I was thinking that targeting an element with the `input[maxlength='22']` selector felt a little dirty 🛀 

No strong opinions on this, changed to using the class name 👍